packages/component-library/draft/Kaizen/GuidanceBlock/GuidanceBlock.scss

```diff
@@ -36,7 +37,7 @@ $ca-banner-collapse-height-delayed: margin-top $ca-duration-fast
   padding: $ca-grid / 2 $ca-grid / 2 0 $ca-grid / 2;
 
   h3 {
-    margin-bottom: 0;
+    margin-bottom: $ca-grid / 2;
```

Comment:
![image](https://user-images.githubusercontent.com/6406263/75316550-c2e1e680-58b9-11ea-8ada-31a5042efc31.png)

The way the Text component is intended to be used is that you pick ` style` prop for your style, and the `tag` prop can be whatever is semantically correct for your use case on the page (i.e. if it's a heading then `h3` is good).

So in your case,
![image](https://user-images.githubusercontent.com/6406263/75317121-40f2bd00-58bb-11ea-9d3a-2607aa80750a.png)
would look like:
```
<Text style="heading" tag="h3">{this.props.text.title}</Text>
<Text style="notification" tag="p">{this.props.text.description}</Text>
// (I think it's notification, could be paragraph though, need to check visually)
```
I just checked our Storybook and realised the stories don't match this, which sucks and I'll aim to change them so they do demonstrate this approach – sorry for any confusion.

For the `margin-bottom`, rather than reaching into the component (which makes the component brittle), we encourage people to wrap the Text component in a div and put padding/margin on that. The upcoming text components will make this much easier by the way!

Reply:
> You can strip out any in-built margins from the Text component by using the inline prop.
Does this also change it do `display: inline` -  I don't know that that's what you'd want?

Reply:
(looking at the component implementation, it adds `inline-flex`... still, I don't know if that's want you want if you just want to remove margins)

Reply:
> Does this also change it do display: inline - I don't know that that's what you'd want?

If you're wrapping it in a `div` to for the margin, and a `div` is `display: block`, the `inline` becomes irrelevant.

Reply:
That makes sense - it's just not super intuitive. All good.

Reply:
Yeah agreed, we're scrapping it in the next batch of text components.

Reply:
what do I need to change in the end? 🤔